epictrack-api/src/api/services/work.py

```diff
@@ -244,6 +246,28 @@ def create_work(cls, payload, commit: bool = True):
             db.session.commit()
         return work
 
+    @classmethod
+    def create_special_fields(cls, work: Work):
+        """Create work special fields"""
+        work.flush()
+        work_epd_special_field_data = {
```

Comment:
You still have to call this method in the create_work service method?

Reply:
Done